### ISSUE TITLE
Support for django-allauth >= 0.62

### ISSUE THREAD

**Issue by espenak** [devilry/devilry-django#1300]
There seems to be a rename of ``allauth.socialaccount.providers.dataporten.views.DataportenAdapter`` in django-allauth 0.62. We probably just need to rename it in ``devilry/devilry_dataporten_allauth/views.py``.

**Comment by espenak** [comment]
Somewhat related to #1299

**Comment by espenak** [comment]
Froze the required version to 0.61.x in 1d942cc0eda3f2f48159e246d58c3e29255e3753 until we fix this.

**Comment by Levijatan** [comment]
Also had to add the setting `ACCOUNT_USER_MODEL_EMAIL_FIELD = ''`. Since allauth now only stores emails in lower case and we store emails in an UserEmail model, I had to set this setting to '' to be able to run the migrations from allauth. Commited in 
deb8603.

**Comment by espenak** [comment]
Also moving this to 6.3 since the general dependency updates forces this update.

**Comment by torgeirl** [comment]
Authentication for Devilry 6.3.2 with `django-allauth[socialaccount]==0.62.*` fails due to an error on [line 39 in devilry_dataporten_allauth/callback.py](https://github.com/devilry/devilry-django/blob/6.3.2/devilry/devilry_dataporten_allauth/callback.py#L39):
```
AttributeError: 'DevilryOAuth2CallbackView' object has no attribute 'get_client'
  File "django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "allauth/socialaccount/providers/oauth2/views.py", line 102, in view
    return self.dispatch(request, *args, **kwargs)
  File "devilry/devilry_dataporten_allauth/callback.py", line 39, in dispatch
    client = self.get_client(request, app)
```

Perhaps the changes in f6059d4 wasn't correct?

**Comment by torgeirl** [comment]
Solved in v6.3.3. :+1:
